quickwit-doc-mapping/src/mapper.rs

```diff
@@ -46,7 +46,7 @@ pub enum SortBy {
     /// Sort by a specific field.
     SortByFastField {
         /// Field to sort by.
-        field_name: String,
+        field: Field,
```

Comment:
we want to keep it a String, because the schema might change.

Reply:
I'm not sure about that. 
I think that the unique identifier of a field would be its id and that's why I like to have a field, because it's just the id.

Another rationale I had on that topic is : the doc mapper is responsible for defining the schema, it's the doc mapper that knows the up-to-date schema. That's why we can safely do the name resolving when building the doc mapper.

It may not be true on an old schema : for example if the schema has changed in the past and that the user rename the field, we will not be able to do the resolving on the old segment schema because the name was different in the past.